projects/dsp-ui/src/lib/viewer/values/date-value/date-value.component.ts

```diff
@@ -76,7 +74,7 @@ export class DateValueComponent extends BaseValueComponent implements OnInit, On
 
         this.commentFormControl = new FormControl(null);
 
-        this.ontologyDateSetting = 'MM.dd.YYYY';
+        this.ontologyDateSetting = 'dd.MM.YYYY';
```

Comment:
Could this be defined via an `@Input` too?

Reply:
if yes, just give it `'dd.MM.YYYY'` as the default value.

Reply:
I don't see why not, changed in 1004d5b